src/server/middleware/helmet.ts

```diff
@@ -60,6 +60,8 @@ const defaultSrc = [
   'https://api-js.mixpanel.com',
   'checkoutshopper-live.adyen.com',
   'checkoutshopper-test.adyen.com',
+  'aff.addreax.com',
+  'js.go2sdk.com/v2/tune.js',
```

Comment:
Sorry i missed this before but these should only be domains and protocols. So `js.go2sdk.com` should be sufficient :)